renovate.json

```diff
@@ -1,5 +1,15 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": ["config:recommended", ":disableDependencyDashboard"],
-  "enabledManagers": ["maven-wrapper", "gradle-wrapper", "nodenv"]
+  "enabledManagers": ["maven-wrapper", "gradle-wrapper", "nodenv", "dockerfile", "regex"],
```

Comment:
Can we use dependabot instead of renovate for the Dockerfile to don’t conflict with the other Dockerfile?

Reply:
I will remove "dockerfile" manager :+1:

Reply:
One issue is that the image and plugin will be updated in separate pull requests, which can lead to compatibility issues (but which is possible even if only the renovate is used).